### ISSUE TITLE
when return form two-page mode, current pagenuber is return to 1, not expedient

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#104]
```
What steps will reproduce the problem?
when return form two-page mode, current pagenuber is return to 1, not expedient

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?
flexpaper1.4

Please provide any additional information below.

```

Original issue reported on code.google.com by `yiditu...@163.com` on 16 Aug 2010 at 9:13

**Comment by GoogleCodeExporter** [comment]
Original comment by `erik.eng...@gmail.com` on 24 Sep 2010 at 11:34
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `erik.eng...@devaldi.com` on 7 Jun 2012 at 10:41
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
will be made available in build 1.5.2
```

Original comment by `erik.eng...@devaldi.com` on 7 Jun 2012 at 10:42
